kvbc/src/categorization/block_merkle_category.cpp

```diff
@@ -158,16 +158,16 @@ std::vector<Buffer> versionedKeys(const std::vector<std::string>& keys, const st
   return versioned_keys;
 }
 
-void putLatestKeyVersion(NativeWriteBatch& batch, const Hash& key_hash, TaggedVersion version) {
-  batch.put(BLOCK_MERKLE_LATEST_KEY_VERSION_CF, key_hash, serializeThreadLocal(LatestKeyVersion{version.encode()}));
+void putLatestKeyVersion(NativeWriteBatch& batch, const std::string& key, TaggedVersion version) {
+  batch.put(BLOCK_MERKLE_LATEST_KEY_VERSION_CF, key, serializeThreadLocal(LatestKeyVersion{version.encode()}));
```

Comment:
Do we need to be concerned about data continuity? 
If we upgrade, and previously wrote hash(X) and now X, do we have a way to access the old data?

Reply:
Yes, we do, I've listed that as an important note in the description. We are going to provide a migration tool for existing blockchains in a separate commit.